package.json

```diff
@@ -161,7 +161,9 @@
     "setupFiles": [
       "jest-canvas-mock"
     ],
-    "setupTestFrameworkScriptFile": "<rootDir>/tests/setup.js",
+    "setupFilesAfterEnv" : [
+      "<rootDir>/tests/setup.js"
+    ],
```

Comment:
🎉

Reply:
Why the change here @lieut-data @saturninoabril ?

Reply:
@bradjcoughlin, this resolves a depreciation warning emitted by Jest. Turns out `setupTestFrameworkScriptFile` only supports one file, so Jest is moving to `setupFilesAfterEnv`.